chapters/5-snippet-information.md

```diff
@@ -53,15 +53,11 @@ Example using document URI:
 
 **5.2.3** Cardinality: Mandatory, one.
 
-**5.2.4** DataFormat: ["DocumentRef-"[idstring]":"] SPDXID
+**5.2.4** Data Format: ["DocumentRef-"[idstring]":"] SPDXID
 
-where `DocumentRef-[idstring]`: is an optional reference to an external
+where `DocumentRef-[idstring]`: is an optional reference to an external SPDX document as described in [section 2.6](2-document-creation-information.md#2.6)
```

Comment:
This and next are just removing the line-breaks that were causing the sentence to be split oddly